src/utils/local-settings.ts

```
@@ -83,37 +136,85 @@ export class ConfigManager {
+    
+    fs.writeFileSync(projectConfigPath, JSON.stringify(defaultConfig, null, 2), {
+      mode: 0o644
+    });
+    
+    this.projectConfigPath = projectConfigPath;
+    this.mergedConfig = null; // Reset cache
   }
 
   public setDefaultModel(model: string): void {
+    if (this.projectConfigPath) {
+      return this.setProjectDefaultModel(model);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)